docs/index.css

```diff
@@ -46,3 +46,8 @@ code {
 .Demo ul {
   margin: var(--space-large);
 }
+
+/* add border between topbar and navbar just in the demo */
```

Comment:
what's the backstory here? I think the demo/docs site should have as few 1-off styles as possible

Reply:
If we don't demo NavBar in the documentation page, then we won't need this.